src/core/client/auth/components/ResetPassword.css

```diff
@@ -0,0 +1,9 @@
+.root {
+  width: 330px;
+  padding-top: calc(5 * var(--spacing-unit));
+}
+
+.footer {
+  padding-top: var(--spacing-unit);
+  width: 100%;
+}
```

Comment:
See above, this is obsolte, once `FormField` is fixed

Reply:
👍